packages/inter-protocol/src/proposals/econ-behaviors.js

```diff
@@ -95,16 +95,25 @@ export const startEconomicCommittee = async (
   },
   { options: { econCommitteeOptions = {} } = {} },
 ) => {
+  const STORAGE_PATH = 'committee';
```

Comment:
This suggests it's the only committee. That's true to some extent, but I wonder to what extent we should commit to that. What do you think about `published.committee.economicGovernance` or just `published.committee1`?

Reply:
Good call. Updated to be `published.committees.Initial_Economic_Committee`, sanitizing the `committeeName` property.